assembl/static2/js/app/components/administration/discussion/personnaliseInterface/index.jsx

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from 'react';
+import { type ApolloClient, withApollo } from 'react-apollo';
+import arrayMutators from 'final-form-arrays';
+import { I18n, Translate } from 'react-redux-i18n';
+import { Field } from 'react-final-form';
+
+import LoadSaveReinitializeForm, { type TInitialValues } from '../../../form/LoadSaveReinitializeForm';
+import SaveButton from '../../saveButton';
+import SectionTitle from '../../sectionTitle';
+import FileUploaderFieldAdapter from '../../../form/fileUploaderFieldAdapter';
+import TextFieldAdapter from '../../../form/textFieldAdapter';
+import { load, postLoadFormat } from './load';
+import { createMutationsPromises, save } from './save';
+import validate from './validate';
+import Loader from '../../../common/loader';
+import { DEFAUT_FAVICON } from '../../../../constants';
+
+type Props = {
+  client: ApolloClient
+};
+
+const loading = <Loader />;
+
+const PersonnaliseInterface = ({ client }: Props) => (
+  <div className="admin-box">
+    <SectionTitle title={I18n.t('administration.discussion.6')} annotation={I18n.t('administration.annotation')} />
+    <div className="admin-language-content">
+      <LoadSaveReinitializeForm
+        load={(fetchPolicy: FetchPolicy) => load(client, fetchPolicy)}
+        loading={loading}
+        postLoadFormat={postLoadFormat}
+        createMutationsPromises={createMutationsPromises(client)}
+        save={save}
+        onSave={(values: TInitialValues) => {
+          // Update the title and the favicon of the page
+          const head = document.head;
+          if (head) {
+            // Update the title
+            const pageTitle = head.getElementsByTagName('title')[0];
+            if (pageTitle) pageTitle.text = values.title;
+            // Update the favicon
+            const favisonLink = head.querySelectorAll('link[rel="icon"]')[0];
```

Comment:
> querySelectorAll

 there can be more than 1 link of this kind ?

Reply:
No, we need a querySelector here +1